src/server.rs

```diff
@@ -392,7 +426,11 @@ mod test {
     }
 
     fn device_id() -> String {
-        thread_rng().gen_ascii_chars().take(8).collect()
+        iter::repeat(())
+            .map(|()| thread_rng().sample(distributions::Alphanumeric))
+            .take(8)
+            .collect()
+        //thread_rng().gen_ascii_chars().take(8).collect()
```

Comment:
Why leave the commented out one?

Reply:
because I forgot to delete it when I was fixing the obsolete call. (argh)